docs/twitchio.rst

```diff
@@ -4,26 +4,25 @@ TwitchIO
 Support
 ---------------------------
 For support using TwitchIO, please join the official `support server
-<http://discord.gg/JhW28zp>`_ on `Discord <https://discordapp.com/>`_.
+<https://discord.gg/RAKc3HF>`_ on `Discord <https://discordapp.com/>`_.
+
+`Become a patreon <https://twitchio.readthedocs.io/en/rewrite/twitchio.html>`_ and help support TwitchIO's development <3.
```

Comment:
```suggestion
`Become a patreon <https://www.patreon.com/twitchio>`_ and help support TwitchIO's development <3.
```

Reply:
That's weird, I swear I copied that one. Thanks for fixing it :)

Reply:
It should also be `patron` instead of `patreon`, seems my edit didn't make it.

Reply:
Sorted and added coauthor manually.